workers/ohsome_quality_analyst/indicators/mapping_saturation/metrics.py

```diff
@@ -0,0 +1,29 @@
+"""Collection of common error metrics as functions."""
+
+import numpy as np
+from numpy import float64, ndarray
+
+
+def me(actual: ndarray, predicted: ndarray) -> float64:
+    """Mean Error"""
+    return np.mean(np.subtract(actual, predicted))
```

Comment:
Not sure what the actualy diagnostic here is.

np.mean returns a float. I checked manually as well as with the docs.
np.substracts works an arraylike objects.

Reply:
> **Returns**    **m :** ***ndarray, see dtype parameter above***
>
> If out=None, returns a new array containing the mean values, otherwise a reference to the output array is returned.

Source: https://numpy.org/doc/stable/reference/generated/numpy.mean.html

Reply:
Remove metrics file since only one metric is used. Implement metric directly in base model class.
 https://github.com/GIScience/ohsome-quality-analyst/pull/170/commits/e8e5df0891672c643733abba7b39d18a03913515